config_app/js/core-config-setup/config-setup-tool.html

```diff
@@ -53,17 +53,25 @@
         </div>
         <div class="co-panel-body">
           <table class="config-table">
+            <tr ng-if="isFieldReadonly('hostname')">
```

Comment:
Use [one-time binding](https://docs.angularjs.org/guide/expression#one-time-binding) to improve performance (`::isFieldReadonly(...)`).